build.proj

```diff
@@ -58,12 +57,44 @@
     <MSBuild Projects="@(_BuildMangedProjects)" Properties="$(ProjectProperties)" />
   </Target>
 
+  <Target Name="BuildTests">
+    <ItemGroup>
+      <_TestProjects Include="src\tests.builds" />
+    </ItemGroup>
+
+    <MSBuild Projects="@(_TestProjects)"
+             Targets="BuildAllProjects"
+             Properties="$(ProjectProperties);BuildTests=true" />
+  </Target>
+
   <Target Name="Test">
     <ItemGroup>
       <_TestProjects Include="src\tests.builds" />
     </ItemGroup>
 
-    <MSBuild Projects="@(_TestProjects)" Properties="$(ProjectProperties)" />
+    <MSBuild Projects="@(_TestProjects)"
+             Targets="TestAllProjects"
+             Properties="$(ProjectProperties);Test=true" />
+  </Target>
+
+  <Target Name="IntegrationTest">
```

Comment:
I guess this target + PerformanceTest is now called directly by arcade due to the script parameters?

Reply:
Yes, these targets are called by arcade same as the build target: https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/Build.proj#L106-L115

Reply:
We are missing the rebuild target which is pretty new. Will do that in another PR.